mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/ListMailboxAssert.java

```diff
@@ -27,48 +27,28 @@
 import org.apache.james.mailbox.store.mail.model.impl.SimpleMailbox;
 
 import com.google.common.base.Function;
+import com.google.common.base.Objects;
 import com.google.common.collect.FluentIterable;
 
 public class ListMailboxAssert {
```

Comment:
It is not possible to define a proper equal on Mailbox?
Else you could use isEqualToComparingOnlyGivenFields, no?